functions/populateCounselors.ts

```diff
@@ -8,34 +8,45 @@ import {
 
 const TokenValidator = require('twilio-flex-token-validator').functionValidator;
 
-const send = (response: TwilioResponse) => (statusCode: number) => (body: string | object) => (
-  callback: ServerlessCallback,
+// TODO: Factor out into lib
+const send = (statusCode: number) => (body: string | object) => (callback: ServerlessCallback) => (
+  response: TwilioResponse,
 ) => {
   response.setStatusCode(statusCode);
   response.setBody(body);
   callback(null, response);
 };
 
-type EventBody = {
+// TODO: Factor out into lib
+const responseWithCors = () => {
+  const response = new Twilio.Response();
+
+  response.setHeaders({
+    'Access-Control-Allow-Origin': '*',
```

Comment:
Unlike the translations this does have info we don't want to openly share, and it'd be nice to start out with it a little more limited than `*`.

Unfortunately [it doesn't look like you can have a list of multiple domains here](https://stackoverflow.com/questions/1653308/access-control-allow-origin-multiple-origin-domains), but you could check the `origin` header in the request and return that domain.  So in bad pseudocode, something like:

```
const origin = context.request.getHeader('origin');
if (['http://localhost:3000', 'https://flex.twilio.com'].includes(origin) {
  response.setHeaders({
    'Access-Control-Allow-Origin': origin,
    ...
  });
} else {
  return no cors headers
};

Please take 15 minutes to see if that works.  If not, we can use `'*'` and create an issue to follow up later.  I realize that using the token validator does protect us, but multiple layers of security aren't a bad idea.